apps/backend/src/main.ts

```diff
@@ -19,6 +22,15 @@ async function bootstrap() {
     })
   );
   app.use(json({limit: '50mb'}));
+  app.use(
+    session({
+      secret: configService.get('JWT_SECRET') || generateDefault(),
```

Comment:
Does the app even start without the JWT? According to the documentation it is a required value so there should be no need for a default.

Reply:
ConfigService will return String or Undefined, so while it won't start without it, it's still required to have a fallback:
![image](https://user-images.githubusercontent.com/66680985/104200704-f86ec000-53f6-11eb-8370-ea4f13bde31b.png)